amazon-kinesis-client/src/main/java/software/amazon/kinesis/coordinator/Scheduler.java

```diff
@@ -235,13 +247,17 @@ public void run() {
 
         try {
             initialize();
-            log.info("Initialization complete. Starting worker loop.");
+            log.info("Initialization complete. Scheduling periodicShardSync..");
+
+            periodicShardSyncManager.start();
```

Comment:
As part of the design requirement. The leader elected shard sync supposed to be the first to be bootstrapped. Should this be in the Scheduler or as part of the bootstrap code.

Reply:
Do you mean the initialize() in Scheduler(see comment above) or is there additional bootstrapping apart from Scheduler?